python/pylibraft/pylibraft/neighbors/ivf_pq/ivf_pq.pyx

```diff
@@ -156,6 +156,14 @@ cdef class IndexParams:
         repeated calls to `extend` (extending the database).
         To disable this behavior and use as little GPU memory for the
         database as possible, set this flat to `True`.
+    max_train_points_per_pq_code : int, default = 256
+        The max number of data points to use per PQ code during PQ codebook
+        training. Using more data points per PQ code may increase the
+        quality of PQ codebook but may also increase the build time. The
+        parameter is applied to both PQ codebook generation methods, i.e.,
+        PER_SUBSPACE and PER_CLUSTER. For PER_CLUSTER,
+        max_train_points_per_pq_code per pq_dim numbers of points would be
```

Comment:
Remove or reformulate the last sentence (same as for C++ docstring).

Reply:
Fixed